addons/website_forum/views/forum_forum_templates_post.xml

```diff
@@ -40,7 +40,7 @@
                 t-value="'/forum/' + slug(question_tag.forum_id) + '/tag/' + slug(question_tag) + '/questions?' + keep_query( 'search', 'sorting', 'my', 'create_uid', filters='tag')"/>
 
             <a t-att-href="click_action"
-                t-attf-class="badge position-relative z-1 #{ 'text-bg-primary' if tag and tag.name == question_tag.name else 'text-bg-secondary' } fw-normal o_tag o_tag_color_#{question_tag.color}"
+                t-attf-class="badge position-relative z-1 fw-normal o_tag o_color_#{question_tag.color}"
```

Comment:
No need to differentiate `question_tag.name` or not anymore?

Reply:
I don't think so, there was a [recently merged PR](https://github.com/odoo/odoo/pull/163763) that aimed to introduce `o_tag_color_*` badges within some modules, but this PR kept the old styling for some reasons (?), which leaves useless classes in the DOM, if I understand correctly. @odoo/rd-sm-learn, could you confirm wether or not this whole part can be removed ? 👀 

<img width="393" alt="image" src="https://github.com/user-attachments/assets/9ef748d5-575a-4774-ab23-61fe7d571580">